### DISCUSSION TITLE
Cannot pull some Ubuntu netboot images

### DISCUSSION THREAD

**Discussion by vawaver** [antonym/netboot.xyz#1401] [General]
Greetings,

I am experiencing difficulty in retrieving (pull selected) these Ubuntu netboot images for local use:

Asset path	
ubuntu-netboot-22.04-amd64	[22.04.3-7e94379b/filesystem.squashfs](https://github.com/netbootxyz/ubuntu-squash/releases/download/22.04.3-7e94379b/filesystem.squashfs)	
ubuntu-netboot-22.04-arm64	[22.04.3-63c09265/filesystem.squashfs](https://github.com/netbootxyz/ubuntu-squash/releases/download/22.04.3-63c09265/filesystem.squashfs)

If I try to download via link than I receive  - Page not found.
![page-not-found](https://github.com/netbootxyz/netboot.xyz/assets/52236863/03d30787-31ce-4b4d-895b-81d6c42eea25)

Could someone please confirm this issue? 
It seems that other Ubuntu 22.04.3 netboot images can be downloaded without any problems.

**Comment by mwd1973** [comment]
I am having the same issue. I believe.
